bitbots_dynup/include/bitbots_dynup/dynup_utils.h

```diff
@@ -2,19 +2,22 @@
 #define BITBOTS_DYNUP_INCLUDE_BITBOTS_DYNUP_DYNUP_UTILS_H_
 
 #include <tf2/LinearMath/Transform.h>
-#include <geometry_msgs/Pose.h>
 
 struct DynupResponse {
-  geometry_msgs::Point support_point;
-  tf2::Transform l_foot_pose;
-  tf2::Transform trunk_pose;
+  tf2::Transform l_foot_goal_pose;
+  tf2::Transform r_foot_goal_pose;
+  tf2::Transform l_hand_goal_pose;
+  tf2::Transform r_hand_goal_pose;
+  bool is_stabilizing_needed;
 };
 
 struct DynupRequest {
-  /* Whether the robot should stand up from the front or back */
-  bool front;
+  /* Whether the robot should stand up from the front, back or from squad */
+  std::string direction;
```

Comment:
yes an enum would be better

Reply:
Agreed with changing it in the engine, but changing the message would be a lot of work and additional code, since ros message enums are not directly compatible with c++ enums and we would also have to touch the hcm to turn the input parameter into an enum (which would require having a third definition of that enum, since that part of the code is python). Since that part of the code is relatively readable as is, I'm not sure whether this would be worth the effort.

Reply:
Are you sure that you cannot use the message's enum definition?
Anyway, we can open an issue to do it later and get the whole thing merged first.